### PR TITLE
Reader: Release A8C Conversations to Prod

### PR DESCRIPTION
**Pull request by blowery**
Only release the a8c stream.

### CODE REVIEW

File: client/reader/sidebar/index.jsx [hunk 1]
```diff
@@ -175,7 +175,8 @@ export const ReaderSidebar = createReactClass( {
 									{ this.props.translate( 'Manage' ) }
 								</a>
 							</li>
-							{ config.isEnabled( 'reader/conversations' ) && (
+							{ config.isEnabled( 'reader/conversations' ) &&
```
Comment (samouri) [inline client/reader/sidebar/index.jsx]: what happens if a user goes directly to the url?

Comment (blowery) [reply client/reader/sidebar/index.jsx]: same thing that happens now if you're on prod

Comment (blowery) [reply client/reader/sidebar/index.jsx]: oh. right. crap.

Comment (blowery) [reply client/reader/sidebar/index.jsx]: hmmmm. looks like we need a sysreq either way to route /read/conversations to prod and not to atlas. 

my gut says you'd get to see the convos tool? we could try to gate it on /read/teams coming back from the sidebar somehow but ... i'm not really that worried about it?

File: client/reader/sidebar/index.jsx [hunk 2]
```diff
@@ -198,8 +197,7 @@ export const ReaderSidebar = createReactClass( {
 								</li>
 							) }
 							<ReaderSidebarTeams teams={ this.props.teams } path={ this.props.path } />
-							{ config.isEnabled( 'reader/conversations' ) &&
-							isAutomatticTeamMember( this.props.teams ) && (
+							{ isAutomatticTeamMember( this.props.teams ) && (
```
Comment (samouri) [inline client/reader/sidebar/index.jsx]: lets double check this default to false when a user doesn't have a cache

Comment (blowery) [reply client/reader/sidebar/index.jsx]: `export const isAutomatticTeamMember = teams => !! find( teams, [ 'slug', 'a8c' ] );`

**Comment by blowery** [comment]
Closing in favor of #18887

**Pull request closed without merging**